clients/go/apiban-ipsets/config.ini

```diff
@@ -0,0 +1,21 @@
+# apiban-ipsets configuration file
+# SSO token
+TOKEN="authtokenstring"
+# lk id
+LKID="0"
+# URL of white/blacklisted entries in DB
+URL="https://siem.intuitivelabs.com/api/"
+# iptables chain name for matching entries
+CHAIN="BLOCKER"
+# interval for the list refresh
+INTERVAL="60s"
+# yes/no - starting from scratch
+FULL="yes"
```

Comment:
what is `FULL` ?

Reply:
`./clients/go/config.go`

```
  Full    string        `long:"FULL" description:"yes/no - starting from scratch"
```
I guess we're not using it?